INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': abb3d4b4-1898-438f-90b9-be029bb4c6ec, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:517 {'message': 'ingesting dataset', 'collection_version_id': abb3d4b4-1898-438f-90b9-be029bb4c6ec, 'url': None, 'current_dataset_version_id': ab629f91-d417-40cd-bc21-7142bf48f2b2}
ERROR    processing:dataset_metadata_update.py:235 Cannot find raw H5AD artifact uri for ab629f91-d417-40cd-bc21-7142bf48f2b2.